src/FSharp.Control.AsyncSeq/AssemblyInfo.fs

```diff
@@ -1,13 +0,0 @@
-﻿namespace System
-open System.Reflection
-
-[<assembly: AssemblyTitleAttribute("FSharp.Control.AsyncSeq")>]
```

Comment:
Atm these `assembly:` attr are not generated. issue is already in vf# repo.
so is better to leave the `AssemblyInfo.fs` in place
The properties in fsproj will populate the nupkg metadata only